Model/Classes/Flight.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AirportTicketBooking.Utils;
+using System.Text;
+
+namespace AirportTicketBooking.Model.Classes;
+#nullable disable
+public class Flight
+{
+    private static long _id = 0;
+    public long Id { get; init; } = ++_id;
```

Comment:
I like how you implement a unique ID, but I have a note that value for ID is not the perfect way to do it because it may face issues in multi-threaded env.